hub/powertoys/always-on-top.md

```diff
@@ -0,0 +1,33 @@
+---
+title: PowerToys Always on Top utility for Windows
+description: A system-wide utility for Windows that allows you to pin windows to the top of your screen. 
+ms.date: 05/28/2021
+ms.topic: article
+ms.localizationpriority: medium
+no-loc: [PowerToys, Windows, Always on Top]
+---
+
+# Always on Top utility
+
+A system-wide utility for Windows that allows you to pin windows to the top of your screen. 
+
+![ColorPicker](../images/pt-always-on-top.png)
+
+## Getting started
+
+### Togglign windows to be on top
+
+With the activation / deactivation shortcut (default: <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>T</kbd>), the window will be toggle on top.  By default, you'll have a 
```

Comment:
- I thought it was "win + ctrl + T" ? (based on [this commit](https://github.com/microsoft/PowerToys/blob/d2de24472235eddb4b35c5a9112520a8409da0af/src/modules/alwaysontop/AlwaysOnTop/Settings.h))
- sentence is truncated

Reply:
you are correct, looks like i had an old setting